f5/bigip/tm/ltm/profile.py

```diff
@@ -56,9 +108,1070 @@ def __init__(self, client_ssls):
         self._meta_data['required_json_kind'] =\
             'tm:ltm:profile:client-ssl:client-sslstate'
 
-    def create(self, certname, keyname):
-        payload = {"name": certname[:-4],
-                   "cert": certname,
-                   "key": keyname}
-        logging.debug(payload)
-        return self._create(**payload)
+    def create(self, **kwargs):
+        """Allows creation of SSL profile with just key/cert names.
+
+            Will require 'name' kwarg to be provided otherwise
+
+        """
+        if 'certname' and 'keyname' in kwargs:
+            certname = kwargs.pop('certname')
+            keyname = kwargs.pop('keyname')
+            kwargs['certKeyChain'] =\
+                [{'cert': certname, 'key': keyname,
+                  'name': certname.split('/')[2][:-4]}]
```

Comment:
why do you even hard code things like certname, keyname? why not just promote proper syntax ? we can have this nice and vanilla like I had initially: where we just used bog standard create method, where only required parameter was 'name:'

This way the user controls the rest